Dockerfile.product

```diff
@@ -39,6 +39,7 @@ FROM openshift3/origin-base
 RUN mkdir -p /opt/bridge/bin
 COPY --from=gobuilder /go/src/github.com/openshift/console/bin/bridge /opt/bridge/bin
 COPY --from=nodebuilder /opt/app-root/src/frontend/public/dist /opt/bridge/static
+COPY --from=build /go/src/github.com/openshift/console/pkg/graphql/schema.graphql /pkg/graphql/schema.graphql
```

Comment:
```suggestion
COPY --from=gobuilder /go/src/github.com/openshift/console/pkg/graphql/schema.graphql /pkg/graphql/schema.graphql
```

Reply:
github is having some issues. I've updated the code a while ago https://github.com/rawagner/console/commit/f91a7abf57a6ec28479cfd820ab94e29b09abf7f but its not reflected here yet. I think we can just omit `--from`

Reply:
yeah I was having issues with commenting on the PR.

> I think we can just omit --from

I would rather follow the instructions that we have in the `Dockerfile` from your previous PR so they dont differ

Reply:
ok, added `--from=gobuilder`